### PR TITLE
esl-popup: create appendArrow

### PR DESCRIPTION
**Pull request by dshovchko**
In the scope of PR:
- moved appendArrow from esl-tooltip up to esl-popup
- made this method public
- created attribute to setup class name of arrow

### CODE REVIEW

File: src/modules/esl-tooltip/core/esl-tooltip.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export class ESLTooltip extends ESLPopup {
 
   public override connectedCallback(): void {
     if (!this.disableArrow) {
```
[the remark this answers is not in the document]
Comment (dshovchko) [reply src/modules/esl-tooltip/core/esl-tooltip.ts:75]: Popup doesn't build its content. It works with what's inside it.

How should it behave if the arrow element is already contained inside?

[the remark this answers is not in the document]
Comment (dshovchko) [reply src/modules/esl-tooltip/core/esl-tooltip.ts:75]: Okay. Suppose the arrow creation is moved to the popup level.
If the arrow element does not exist in the popup children, the popup constructor will create it.

But there is a question about the `disable-arrow` attribute. Right now it prevents the arrow element from being created on the tooltip. But after it is moved to the popup level how should the popup respond to its presence? Not to create the arrow? But it seems more obvious to the user that the popup won't have an arrow element at all. It means that if the popup has a `disable-arrow` option but the arrow is manually inserted into the layout then we should disregard it.

Would it be enough to rely on styles?
```
esl-popup[disable-arrow] .esl-popup-arrow {
  display: none;
}
```

Or should we force the arrow elements to `display: none`?

[the remark this answers is not in the document]
Comment (ala-n) [reply src/modules/esl-tooltip/core/esl-tooltip.ts:75]: Let's also back to discussion on our regular meeting on Monday

**Comment by ala-n** [comment]
Discussion: 
1) No arrow in DOM + `disable-arrow` (active)  -> 
  - No arrow creation
  - Be aware of the required attribute observation: removing `disable-arrow` leads to arrow creation
2) No arrow in DOM + no `disable-arrow` (inactive)  ->
  - create an arrow automatically
3) Aroow presented in the  DOM + any state of  `disable-arrow` attribute
  - if  `disable-arrow` active  -> optional CSS style to hide (by default)
  
  
  Overall cases:
  1 As defined above
  2 invert logic, attribute `create-arrow`
  3 always auto-create arrow, positioning correctly handle `display: none` from styles anyway  <-
  
  --- 
  Final: 
  1) Autocreate arrow on request access `this.$arrow` (get or create)
  2) `disable-arrow` (at least for the 4th version) -> CSS only attribute to disable arrow

File: src/modules/esl-popup/core/esl-popup.ts [hunk 2]
```diff
@@ -111,6 +113,12 @@ export class ESLPopup extends ESLToggleable {
   @prop() public override closeOnEsc = true;
   @prop() public override closeOnOutsideAction = true;
 
+  /** Arrow element */
+  @memoize()
+  public get $arrow(): HTMLElement | null {
+    return this.querySelector(`span.${this.arrowClass}`) || this.appendArrow();
```
Comment (ala-n) [inline src/modules/esl-popup/core/esl-popup.ts:119]:
```suggestion
    return this.querySelector(`.${this.arrowClass}`) || this.appendArrow();
```

**Review by ala-n — APPROVED** [review]

**Review by julia-murashko — APPROVED** [review]

**Review by NastaLeo — APPROVED** [review]

**Pull request merged**